pages/linux/systemctl.md

```diff
@@ -18,6 +18,10 @@
 
 `systemctl enable/disable {{unit}}`
 
+- Mask/Unmask a unit, make it impossible to start it:
```

Comment:
In that case, I'd suggest changing the description to make it clear this is about preventing automatic execution, rather than preventing execution altogether. Or did I get that wrong?